packages/dht/src/dht/IPeerManager.ts

```diff
@@ -0,0 +1,13 @@
+import { PeerIDKey } from '../helpers/PeerID'
+import { PeerDescriptor } from '../proto/packages/dht/protos/DhtRpc'
+import { DhtPeer } from './DhtPeer'
+
+export interface IPeerManager {
```

Comment:
Are we planning that there will be multiple `PeerManager` implementations? Or why an interface is needed?

Reply:
Yes, the plan is to have multiple ones at least in the development phase, and to compare which implementations give the best benchmark results.

Reply:
Maybe good to add a comment about that fact, and `TODO` comment so that we can remove the interface when the development of this feature has been completed.